frontend/packages/noobaa-storage-plugin/src/components/object-data-reduction-card/object-data-reduction-card-item.tsx

```diff
@@ -2,85 +2,60 @@ import * as React from 'react';
 import * as _ from 'lodash';
 import { DashboardCardHelp } from '@console/internal/components/dashboard/dashboard-card';
 import {
-  humanizeBinaryBytesWithoutB,
+  humanizeBinaryBytes,
```

Comment:
Is it fine to change it here ?
#2296 is going in or not ?
@cloudbehl

Reply:
@afreen23 I don't think so.